gatling-jms/src/main/scala/io/gatling/jms/check/JmsJsonPathCheckMaterializer.scala

```diff
@@ -20,24 +20,27 @@ import com.fasterxml.jackson.databind.JsonNode
 import io.gatling.commons.validation._
 import io.gatling.core.check.jsonpath.JsonPathCheckType
 import io.gatling.core.check.{ CheckMaterializer, Preparer }
+import io.gatling.core.config.GatlingConfiguration
 import io.gatling.core.json.JsonParsers
 import io.gatling.jms.JmsCheck
+import io.gatling.jms.check.JmsMessageBodyPreparers.toBytes
 import javax.jms.{ BytesMessage, Message, TextMessage }
 
-class JmsJsonPathCheckMaterializer(jsonParsers: JsonParsers) extends CheckMaterializer[JsonPathCheckType, JmsCheck, Message, JsonNode](identity) {
+class JmsJsonPathCheckMaterializer(jsonParsers: JsonParsers)(implicit config: GatlingConfiguration)
```

Comment:
Could you please pass the config explicitly here? We try to limit implicits to the DSL layer.